cli/types/cypress.d.ts

```diff
@@ -2352,9 +2352,14 @@ declare namespace Cypress {
      * @default "spec"
      */
     reporter: string
+    /**
+     * Some reporters accept [reporterOptions](https://docs.cypress.io/guides/tooling/reporters.html#Reporter-Options) that customize their behavior
```

Comment:
Use quicklink as this link could go stale

```suggestion
     * Some reporters accept [reporterOptions](https://on.cypress.io/reporters) that customize their behavior
```